index_hash.txt

```diff
@@ -1 +1 @@
-46f4dcae39877158b31cc736c8dd32e11ef521775d386e13ab8523080cab05ca
\ No newline at end of file
+3cabb4c1087b654754313ec0113e05b95ed6cb211a4d6e15843d8a38c360ff5f
```

Comment:
Do we still need the hash? I thought we'd switched to using the git hash to check if we need to reload the file?

Reply:
@blckmn proposed to use https://api.betaflight.com/presets/** that will be automatically updating with every change in the git. IN this case I guess we still need index_hash.txt. But I am not using it yet and maybe won't at the begining.